vendor.conf

```diff
@@ -42,3 +42,33 @@ golang.org/x/text 19e51611da83d6be54ddafce4a4af510cb3e9ea4
 github.com/dmcgowan/go-tar go1.10
 github.com/stevvooe/ttrpc d4528379866b0ce7e9d71f3eb96f0582fc374577
 github.com/syndtr/gocapability db04d3cc01c8b54962a58ec7e491717d06cfcc16
+github.com/containerd/cri-containerd 83573155645882403ef2da86df8ea95e2d031f28
```

Comment:
Can we separate this with `#` sections?  cri-containerd is fine there but then can we make a section for 

```
#cri dependencies
gihub.com/blang/...
```

I will help us know where these things came from.

Reply:
Will do.

Reply:
Done.